lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -616,6 +623,42 @@ function dosomething_reportback_view_entity($entity) {
   return theme('reportback_permalink', $permalink_vars);
 }
 
+/**
+ * Callback for /reportback/failure page.
+ *
+ * We hit this when a reportback sent to Rogue fails to send back to Phoenix
+ */
+function dosomething_reportback_view_failure() {
+  // Get the user in all the forms that we need
+  global $user;
+  $rb_simple_user = dosomething_user_get_simple_user($rb_user = user_load($user));
+
+
+  // Load the campaign node so we can find out its name, if there's a scholarship, etc.
+  $nid = $_GET['nid'];
+  $node = dosomething_campaign_load(node_load($nid));
+
+  // Get all the copy and allow admins to build a link to the user's profile
+  $copy_vars = _dosomething_reportback_get_permalink_copy_vars($rb_user);
+  $failure_copy = token_replace(t(variable_get('dosomething_campaign_permalink_failure_copy', '')), ['user' => $user]);
+  $incentive = ($node->scholarship) ? $copy_vars['owners_rb_scholarship'] : "";
+
+  // Get the image
+  $file = reportback_file_load(variable_get('dosomething_campaign_permalink_failure_image_fid'));
```

Comment:
I think this probably throws a lot of errors because it's trying to load an actual reportback item, but we are just using an image node to use for this, right? 
should be a call to `dosomething_image_get_themed_image`